eidangservices/federator/server/routes/dataselect.py

```diff
@@ -85,6 +91,10 @@ def get(self, args, stream_epochs):
         ManyStreamEpochSchema(context={'request': request}),
         locations=('form',)
     )
+    @with_strict_args(
+        (DataselectSchema(), StreamEpochSchema()),
```

Comment:
For **POST** requests the configuration of `@with_strict_args` with a `StreamEpochSchema` instance is incorrect. Within a *postfile* exclusively `keyword=value` pairs (from the `DataselectSchema`) and *SNCL lines* are allowed.

To be corrected for the other resources.